scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1540,13 +1552,32 @@ public String getTaskServerLogsByTag(String jobId, String taskTag)
         if (myJobsOnly) {
             user = ident.getUsername();
         }
-        return dbManager.getJobs(offset,
-                                 limit,
-                                 user,
-                                 filterCriteria.isPending(),
-                                 filterCriteria.isRunning(),
-                                 filterCriteria.isFinished(),
-                                 sortParameters);
+
+        Page<JobInfo> jobsInfo = dbManager.getJobs(offset,
+                                                   limit,
+                                                   user,
+                                                   filterCriteria.isPending(),
+                                                   filterCriteria.isRunning(),
+                                                   filterCriteria.isFinished(),
+                                                   sortParameters);
+
+        /**
+         * Add/inject to each JobInfo the list of signals, if they exist, used by the job.
```

Comment:
Add/inject to each JobInfo the list of signals, if they exist, used by the job => Add/inject to each JobInfo the list of signals used by the job, if they exist 
is easier to read

Reply:
yes